cmd/manager/main.go

```
@@ -143,14 +150,40 @@ func main() {
 		os.Exit(1)
 	}
 
-	baseStorageURL, err := url.Parse(fmt.Sprintf("%s/catalogs/", httpExternalAddr))
+	if certFile != "" && keyFile != "" {
```

I think if only one is set, we should error out instead of silently ignoring the one flag that was set.